sdk/servicefabric/Azure.ResourceManager.ServiceFabric/api/Azure.ResourceManager.ServiceFabric.netstandard2.0.cs

```diff
@@ -54,135 +54,136 @@ public partial class ApplicationResourceData : Azure.ResourceManager.Models.Trac
-        public string ClusterId { get { throw null; } }
-        public Azure.ResourceManager.ServiceFabric.Models.ClusterState? ClusterState { get { throw null; } }
+        public System.Guid? ClusterId { get { throw null; } }
+        public Azure.ResourceManager.ServiceFabric.Models.ServiceFabricClusterState? ClusterState { get { throw null; } }
         public Azure.ResourceManager.ServiceFabric.Models.DiagnosticsStorageAccountConfig DiagnosticsStorageAccountConfig { get { throw null; } set { } }
         public Azure.ETag? ETag { get { throw null; } }
-        public bool? EventStoreServiceEnabled { get { throw null; } set { } }
         public System.Collections.Generic.IList<Azure.ResourceManager.ServiceFabric.Models.SettingsSectionDescription> FabricSettings { get { throw null; } }
-        public bool? InfrastructureServiceManager { get { throw null; } set { } }
+        public bool? IsEventStoreServiceEnabled { get { throw null; } set { } }
+        public bool? IsInfrastructureServiceManagerEnabled { get { throw null; } set { } }
+        public bool? IsWaveUpgradePaused { get { throw null; } set { } }
         public string ManagementEndpoint { get { throw null; } set { } }
         public long? MaxUnusedVersionsToKeep { get { throw null; } set { } }
-        public System.Collections.Generic.IList<Azure.ResourceManager.ServiceFabric.Models.NodeTypeDescription> NodeTypes { get { throw null; } }
-        public System.Collections.Generic.IList<Azure.ResourceManager.ServiceFabric.Models.Notification> Notifications { get { throw null; } }
-        public Azure.ResourceManager.ServiceFabric.Models.ProvisioningState? ProvisioningState { get { throw null; } }
-        public Azure.ResourceManager.ServiceFabric.Models.ReliabilityLevel? ReliabilityLevel { get { throw null; } set { } }
-        public Azure.ResourceManager.ServiceFabric.Models.CertificateDescription ReverseProxyCertificate { get { throw null; } set { } }
-        public Azure.ResourceManager.ServiceFabric.Models.ServerCertificateCommonNames ReverseProxyCertificateCommonNames { get { throw null; } set { } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.ServiceFabric.Models.ClusterNodeTypeDescription> NodeTypes { get { throw null; } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.ServiceFabric.Models.ClusterNotification> Notifications { get { throw null; } }
+        public Azure.ResourceManager.ServiceFabric.Models.ServiceFabricProvisioningState? ProvisioningState { get { throw null; } }
+        public Azure.ResourceManager.ServiceFabric.Models.ClusterReliabilityLevel? ReliabilityLevel { get { throw null; } set { } }
+        public Azure.ResourceManager.ServiceFabric.Models.ClusterCertificateDescription ReverseProxyCertificate { get { throw null; } set { } }
+        public Azure.ResourceManager.ServiceFabric.Models.ClusterServerCertificateCommonNames ReverseProxyCertificateCommonNames { get { throw null; } set { } }
```

Comment:
Should these types also prepend the ServiceFabric prefix? I believe other RPs might also have Cluster and similar things

Reply:
I was struggled this for long time, if all `Cluster` need prepend `ServiceFabric` then should this change be applied to all `Application` and `Service` models / constants, almost all will be changed, and some name will be extremely long.
So here I search these name in all RPs, not find dups, so just keep the sort `Cluster` prefix.